code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -120,6 +120,20 @@
 		new /obj/item/cell/medium/high(src)
 		new /obj/item/storage/pouch/tubular(src)
 
+/obj/item/storage/box/bs_kit/lascore
+	name = "\improper Lascore Primary Kit"
```

Comment:
```suggestion
	name = "\improper Standard Primary Kit"
```

Reply:
No. This instantly makes it less clear what it';s supposed to be in its description of the kit.